styles/cspace/Filter.css

```diff
@@ -57,6 +57,7 @@
 
 .common li input:checked + div::before {
   background-image: url(../../images/check.svg);
+  border: 1px solid inputBorderColor;
```

Comment:
This is not needed, since you changed `inputBorderColor`. It is redundant with this rule:  https://github.com/collectionspace/cspace-public-browser.js/blob/584454cdd2b16edac3ba4ac470ccbab3ef121c61/styles/cspace/Filter.css#L47